app.js

```diff
@@ -90,6 +90,7 @@ app.post('/data/acceptedcategories/organization/:id', data.acceptedCategoriesOrg
 app.get('/data/categories', data.categoriesGet);
 app.get('/data/member', data.member);
 app.get('/data/member/organization/:id', data.organizationMemberGet);
+app.get('/data/organization/member/:id', data.memberOrganizationGet);
```

Comment:
Since `organizationMemberGet` and `memberOrganizationGet` are pretty similar, it might be useful to rename these functions to something more apparent.